### PR TITLE
Update Marketo User Audit Data Stream Docs for event structure change.

### PR DESCRIPTION
**Pull request by guptapraveen2**
CET-1021

Event structure change, wrapped event details in the body element.
Remove the user field.
Add the userId to support easy track of the user details.

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by guptapraveen2** [comment]
@bejohnso I added the equivalent changes in this repo based on feedback from @rperelma

 @kgera, please let me know if any else I need to do before merging these changes.

**Review by rperelma — APPROVED** [review]
LGTM, you can merge

**Comment by guptapraveen2** [comment]
@kgera, I don't have write access to merge. Would you please help me to merge this pull request?

**Pull request merged**
